CMSgov/mdct-mcr
# fix program name in header

## Description

The program name wasn't appropriately setting in the header. Now it is.

### How to test

Open a report in /mcpar/whatever and see that the program name now shows in the header.

### Changed Dependencies

n/a

## Code author checklist

## Reviewer checklist (two different people)
